### PR TITLE
Transaction info & cryptoNews

### PR DESCRIPTION
**Pull request by yaroshchuk8**

### CODE REVIEW

**Review by ILabiak — APPROVED** [review]
Everything's good

File: src/crypto.js [hunk 1]
```diff
@@ -80,85 +80,45 @@ class RawCrypto {
   async cryptoNews() {
     const info = await safeGet(`${this.defaultUrl}/v2/news/?lang=EN`);
     const data = info.Data;
+    const amountOfArticles = 5;
```
Comment (akaeyuhi) [inline src/crypto.js]: Add this variable to arguments maybe

File: src/crypto.js [hunk 2]
```diff
@@ -80,85 +80,45 @@ class RawCrypto {
...
     let bool = true;
     while (bool) {
       console.log(proposedTitles);
       const writtenTitleNumber = await promised.question(
         'Enter number of article\'s title you\'d like to read:\n'
       );
+
+      if (!arrayOfArticleNumbers.includes(+writtenTitleNumber)) {
+        console.log(`${promised.colors.red}Wrong number
+        ${promised.colors.green}`);
```
Comment (akaeyuhi) [inline src/crypto.js]: Add escape query for reset to colors

File: src/exchanges.js [hunk 3]
```diff
@@ -154,6 +154,80 @@ const feesRate = async () => {
...
+  ];
+  const choiceNumbers = [
+    1,
+    2,
+    3,
+    4,
+  ];
+
+  console.log('\nList of cryptos:');
+  for (let i = 0; i < cryptoNames.length; i++) {
```
Comment (akaeyuhi) [inline src/exchanges.js]: Use for of

File: src/exchanges.js [hunk 4]
```diff
@@ -154,6 +154,80 @@ const feesRate = async () => {
...
+    console.log(`${promised.colors.red}Wrong hash${promised.colors.green}`);
+    return;
+  } else if (Object.prototype.hasOwnProperty.call(info, 'confirmed')) {
+    keys.push('confirmed');
+    outputKeys.push('Confirmed at');
+  } else {
+    console.log('\nTransaction isn\'t confirmed yet :C');
+  }
+
+  for (let i = 0; i < keys.length; i++) {
```
Comment (akaeyuhi) [inline src/exchanges.js]: Same problem, use for of here

File: src/exchanges.js [hunk 5]
```diff
@@ -154,6 +154,80 @@ const feesRate = async () => {
   return res;
 };
 
+const transactionInfo = async () => {
```
Comment (akaeyuhi) [inline src/exchanges.js:157]: Try to decompose this method, function is too big

File: src/exchanges.js [hunk 6]
```diff
@@ -154,6 +154,80 @@ const feesRate = async () => {
...
+    'Dogecoin',
+    'Litecoin',
+  ];
+  const abbreviation = [
+    'btc',
+    'dash',
+    'doge',
+    'ltc',
+  ];
+  const choiceNumbers = [
```
Comment (akaeyuhi) [inline src/exchanges.js]: This array is like used nowhere

File: src/crypto.js [hunk 7]
```diff
@@ -80,85 +80,45 @@ class RawCrypto {
   async cryptoNews() {
     const info = await safeGet(`${this.defaultUrl}/v2/news/?lang=EN`);
     const data = info.Data;
+    const amountOfArticles = 5;
+    const arrayOfArticleNumbers = [
```
Comment (akaeyuhi) [inline src/crypto.js]: This array is also unused

File: src/crypto.js [hunk 8]
```diff
@@ -80,85 +80,45 @@ class RawCrypto {
...
+      1,
+      2,
+      3,
+      4,
+      5,
+    ];
 
     let proposedTitles = '\nFive most recent articles on cryptocurrency:\n';
-    for (let i = 0; i < 5; i++) {
+    for (let i = 0; i < amountOfArticles; i++) {
```
Comment (akaeyuhi) [inline src/crypto.js]: You can use Array.prototype.filter instead

**Review by akaeyuhi — CHANGES_REQUESTED** [review]
This code still needs some minor improvements, but overall you fixed problem with magic numbers

**Review by akaeyuhi — APPROVED** [review]
Norm vrode, mozhesh mergit`

**Pull request merged**